.travis.yml

```diff
@@ -54,8 +54,9 @@ before_script:
   - emulator -avd test -no-audio -no-window &
   - android-wait-for-emulator
   - adb shell input keyevent 82 &
+  - export CLOUDINARY_URL=$(bash tools/get_test_cloud.sh);
 
-script: ./gradlew clean connectedCheck
+script: ./gradlew clean connectedCheck -DCLOUDINARY_URL=${CLOUDINARY_URL}
```

Comment:
looks redundant, environment variable is consumed in the code.

Reply:
Removed